### ISSUE TITLE
support multiple screens

### ISSUE THREAD

**Issue by juancarlosfarah** [graasp/graasp-lab-light-pollution-simulator#40]
Currently most elements, like the moon, house, lamp posts are being positioned without thinking about different screens. We need to ensure that most screen dimensions are supported in order to avoid elements rendering outside the canvas or in strange positions.

**Comment by NunonuN** [comment]
I agree. I'm thinking about making all positions relative to dimensions of the window. This is the same approach you used for the lamp-posts, right?

**Comment by juancarlosfarah** [comment]
@NunonuN, yes indeed. The simplest idea would be to make the x and y positions dependent on the sizes of the x and y axes, i.e. the height and width of the window. Once we decide on a minimum x and y to support, we can also scale the objects accordingly and show/hide some depending on the size of the screen. We can do this element by element until we get all of the right. :-)
